### ISSUE TITLE
BREAK Function & NUM Indicator are Disabled

### ISSUE THREAD

**Issue by rruhle** [taviso/123elf#102]
Function Disabled
Break (Ctrl-F12) function wont interrupt macro STEP function
F1 Help Function / Keyboard Index /  Special Keys / Break
F1 Help Function / Keyboard Index / View default keymap (Ctrl-F12)
A few iterations of Break key (Ctrl-F12) should halt macro STEP in progress

Status Indicator Missing
F1 Help Function / Status Indicators /  NUM 
Num Lock works, but the Indicator is missing at bottom of screen

**Comment by taviso** [comment]
Thanks for the report, regarding break: It does work, it's just slightly different on UNIX - In UNIX Ctrl-C - which generates an interrupt signal - is more common! I notice it also does correctly show the break location, so it's weird it doesn't show it while stepping normally. This seems to confirm #101 is a bug we can fix.

Regarding the NUM indicator: You're right, I think that must be an error in the help, there's no way that could have worked on UNIX! 

DOS and UNIX work differently here, In UNIX you read a stream of input events, but toggling num lock doesn't generate any input, it just changes any subsequent input that arrives.

We need some better tools to update and fix the documentation, but they don't exist yet. It would also be nice if we could import the documentation from R4 in DOS, I think they improved some of the `@function` documentation!

**Comment by rruhle** [comment]
Yes I see. (Ctrl-C) interrupts Macro STEP perfectly, just like (Ctrl-Break) interrupt in legacy DOS.

Thanks for addressing NUM Lock function in Unix.

**Comment by rruhle** [comment]
Manually updated Help file for default 123-Keymap.odt
 / Keyboard Index / View defaut keymap
[123-Keymap.odt](https://github.com/taviso/123elf/files/9354978/123-Keymap.odt)
